packages/itwinui-react/src/core/Checkbox/Checkbox.test.tsx

```diff
@@ -115,6 +114,7 @@ it.each(['label', 'input'] as const)(
     const { container } = render(
       <Checkbox
         label={el === 'label' ? 'Some label' : undefined}
+        wrapperProps={{ style: { color: 'blue' } }}
```

Comment:
I think instead of having this `it.each` statement, this test can be written similarly to how your test for `className` is written. Just apply different stylings to `wrapperProps`, `labelProps`, and the input, and check for those three stylings within this test.

It might even be possible to test for `style` and `className` within the same test labeled `'correctly passes props through wrapperProps and labelProps`.

Also, I would make sure you apply different colors to each style instead of `blue` across the board.

Reply:
Done!